src/Dialogs/ExportDialog.vala

```diff
@@ -94,13 +93,15 @@ public class ExportDialog : Gtk.Dialog {
             ctr++;
         }
 
-        pixels_entry = new Gtk.Entry ();
-        pixels_entry.max_length = 6;
-        pixels_entry.text = "%d".printf (current_scale);
-        pixels_entry.xalign = 1;
+        pixels_entry = new Gtk.SpinButton.with_range (1, 999999, 1);
+        pixels_entry.set_max_length (6);
+        pixels_entry.set_value (current_scale);
+        pixels_entry.set_digits (0);
 
-        Gtk.Label pixels_label = new Gtk.Label.with_mnemonic (_("_pixels"));
-        pixels_label.mnemonic_widget = pixels_entry;
+        Gtk.Label size_label = new Gtk.Label.with_mnemonic (_("_Size in pixels:"));
+        size_label.halign = Gtk.Align.END;
+        quality_label.use_underline = true;
```

Comment:
Is this a copy/paste mistake or is this property just in the wrong spot :)

Reply:
Oops, copy/past mistake